app/styles/app.css

```diff
@@ -58,3 +55,11 @@ body {
   align-items: center;
   font-size: 28px;
 }
+
+
+@media (max-width: 460px) {
+  .footer {
+    padding: 8px 8px 16px 8px;
```

Comment:
can this be changed to `8px 8px 16px`, because you are providing the same padding for left and right